doc/CMakeLists.txt

```diff
@@ -86,3 +86,32 @@ add_dependencies(doc open62541)
 
 set_target_properties(doc doc_latex doc_pdf PROPERTIES EXCLUDE_FROM_DEFAULT_BUILD TRUE)
 set_target_properties(doc doc_latex doc_pdf PROPERTIES FOLDER "open62541/doc")
+
+if (UA_PACK_DEBIAN)
+    set(open62541_install_debdoc_html_dir share/doc/open62541/open62541.html)
+    set(open62541_install_debdoc_dir share/doc/open62541)
+
+    set(open62541_debdoc_pdf_files "${DOC_LATEX_DIR}/open62541.pdf")
+
+    install(DIRECTORY ${DOC_HTML_DIR}/
+        COMPONENT htmldoc 
+        DESTINATION ${open62541_install_debdoc_dir}/open62541.html 
+        USE_SOURCE_PERMISSIONS
+        FILES_MATCHING 
+        PATTERN "*.*"
+        PATTERN "_sources" EXCLUDE
+        PATTERN "_static/ua-wireshark-pubsub.png" EXCLUDE
```

Comment:
Why do we need to exclude all these files?
What happens if the doc is updated and more images are added? Do we need to update this list then too?
I fear that this will lead to inconsistencies, since nobody will look into that CMakeLists.txt when changing the docs...

Can we somehow make this more generic?

Reply:
Maybe using a few specific patterns to include files would make it shorter and more generic?

Reply:
There are a lot of leftovers after `cmake/sphinx` which should be definitely removed before installation of the docs.
It seems some excludes are not needed anymore because `dh_sphinxdoc` already does the job.

So we can reduce the excludes to 
```
        PATTERN "_sources" EXCLUDE
        PATTERN "CmakeFiles" EXCLUDE
```
This directories definitely never needed (as far as I can see) for the installed docs.
 I will update the PR ...